releases.yml

```diff
@@ -1,4 +1,11 @@
 releases:
+  4.15.0:
+    assembly:
+      basis:
+        assembly: rc.8
+      group:
+        upgrades!: 4.14.14,4.15.0-ec.0,4.15.0-ec.1,4.15.0-ec.2,4.15.0-ec.3,4.15.0-rc.0,4.15.0-rc.1,4.15.0-rc.2,4.15.0-rc.3,4.15.0-rc.4,4.15.0-rc.5,4.15.0-rc.7
```

Comment:
we also want to include rc.8 in this list, so we get rc.8 -> GA update recommendations.